spec/sidekiq_unique_jobs/sidekiq_worker_methods_spec.rb

```diff
@@ -61,4 +61,22 @@ def initialize(worker_class)
       end
     end
   end
+
+  describe "#worker_options" do
+    subject(:worker_class_constantize) { worker.worker_options }
+
+    let(:worker_class) { UniqueJobOnConflictHash }
+
+    it {
+      is_expected.to eq({
+        "lock" => :until_and_while_executing,
+        "on_conflict" => {
```

Comment:
I think if the rest of the test suite passes, this test should be good enough! 👍